### ISSUE TITLE
[Bug]Different review info of parent and sub-decks

### ISSUE THREAD

**Issue by PANG-hans** [ankidroid/Anki-Android#8691]
It have some same place with https://github.com/ankidroid/Anki-Android/issues/8660, but some differents.
<img height = "400" src = "https://user-images.githubusercontent.com/49405752/115998265-07c9f000-a619-11eb-8cc6-76d47e315fec.jpg">

Step:

1. review `forth` from `14-0-0` to `5-4-0`,
2. review `sec` from `5-0-0` to `2-3-0`,
3. review `thir` from `2-0-0` to `0-0-0`,
4. add 5 new card in `thir`, review all.
5. review `forth` from `0-4-0` to `0-0-0`,
6. Parent deck has `0-4-0` and subdeck `sec` shows `0-3-0`.

Different:
It exist even i do a sync. keeps along

###### Expected Result

Both parent decks and sub deck `sec` shows `0-3-0`.

###### Actual Result

parent decks: `0-4-0`
sub decks: `0-3-0`

###### Debug info

AnkiDroid Version = 2.15alpha28

Android Version = 10

ACRA UUID = b5870e31-45d2-4fbd-ae11-d6d6966052f6

Scheduler = std2

Crash Reports Enabled = false

DatabaseV2 Enabled = true

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by PANG-hans** [comment]
If it received could i work on it?

**Comment by mikehardy** [comment]
> AnkiDroid Version = 2.15alpha28

We are on 2.15alpha45 now https://github.com/ankidroid/Anki-Android/releases - Please make sure you can reproduce this against current code

How does this behave in Anki Desktop? It may be that is the way it is supposed to work - if you can reproduce it there, then it's possibly intentional behavior

**Comment by PANG-hans** [comment]
I tested it on `2.15alpha45`. 
Found the reason:
some cards is stored in default sub-deck, it is not included in any named sub-decks.
But the card broswer in AnkiAndroid can just show all cards when select parent-deck, can not show cards in defalut sub-decks.
The Anki Desktop can not just show cards in defalut sub-decks, but it can change display order to put these cards on top, what is **not** supported in AnkiAndroid. I have created a new issue https://github.com/ankidroid/Anki-Android/issues/8869 .
